graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/GraphQLJpaQueryDataFetcher.java

```diff
@@ -46,11 +46,21 @@
  *
  */
 class GraphQLJpaQueryDataFetcher extends QraphQLJpaBaseDataFetcher {
+
+    private static boolean defaultDistinct = false;
 	
     public GraphQLJpaQueryDataFetcher(EntityManager entityManager, EntityType<?> entityType) {
         super(entityManager, entityType);
     }
 
+    public static boolean isDefaultDistinct() {
+        return defaultDistinct;
+    }
+
+    public static void setDefaultDistinct(boolean defaultDistinct) {
```

Comment:
@miklemv I wonder if it would be better to inject default distinct setting via `GraphQLJpaSchemaBuilder.setDefaultDistinct(..)` method to allow flexible customization per builder instance rather than a hack using static method.